redaxo/src/core/lib/login/webauthn.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+use lbuchs\WebAuthn\Binary\ByteBuffer;
+use lbuchs\WebAuthn\WebAuthn;
+use lbuchs\WebAuthn\WebAuthnException;
+
+/**
+ * @package redaxo\core\login
+ *
+ * @internal
+ */
+class rex_webauthn
```

Comment:
Aktuell habe ich es als `@internal`-Api und spezifisch für das Backend und die Backend-User umgesetzt (entsprechend auch nur in `rex_backend_login`, nicht in der Basis-Klasse `rex_login`.

Hat den Vorteil, dass wir es noch beliebig ändern können.

Aber bin am überlegen, ob man es nicht doch direkt als Public-Api umsetzt, und nutzbar auch im Frontend (somit abstrahiert von `rex_user` und in der Basis-Login-Klasse `rex_login` integriert).

Meinungen dazu? Direkt so umsetzen, oder eher erstmal internal nur fürs Backend belassen?

Reply:
Ich würde erstmal nur backend machen